pkgs/by-name/tl/tlmi-auth/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, meson
+, ninja
+, openssl
+, cmake
+}:
+let
+  name = "tlmi-auth";
+  version = "1.0.1";
+in
+stdenv.mkDerivation {
+  pname = name;
+  version = version;
+
+  src = fetchFromGitHub {
+    owner = "lenovo";
+    repo = name;
+    rev = "v${version}";
+    hash = "sha256-/juXQrb3MsQ6FxmrAa7E1f0vIMu1397tZ1pzLfr56M4=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
+    openssl
+  ];
+
+  meta = with lib; {
```

Comment:
I'm not familiar with the tool but if it has a main executable then adding `mainProgram` would be nice, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Thanks for advice. The build results includes only one binary.  The `mainProgram` meta-attribute is defined now.